package.json

```diff
@@ -17,7 +17,9 @@
     "lerna-version": "lerna version --no-git-tag-version --force-publish",
     "prelerna-publish": "lerna run build",
     "lerna-publish": "lerna publish from-package --no-push --force-publish --no-verify-access --no-verify-registry",
-    "lint": "yarn eslint && yarn stylelint",
+    "lint": "yarn eslint && yarn stylelint && yarn prettier-check",
+    "prettier-check": "yarn prettier --check . --ignore-path .gitignore",
+    "prettier-write": "yarn prettier --write . --ignore-path .gitignore",
```

Comment:
where is this `--write` command used?  ad hoc when a developer wants to write prettier changes to disk without creating a new commit?

I'm also curious how the `--check` command is intended to be used? I would have expected to see that as part of the `lint-staged` config but it appears to be using `--write` instead?

Reply:
> where is this --write command used? ad hoc when a developer wants to write prettier changes to disk without creating a new commit?

That is correct!

> I'm also curious how the --check command is intended to be used? I would have expected to see that as part of the lint-staged config but it appears to be using --write instead?

CI will use `--check`, we could also have it used by lint-staged, Implementations I saw have it using the write flag. Im not strongly opinionated here. I liked that it would autofix issues for our developers so that they could then just go commit the fix instead of manually run a command -- if you have a preference, I'm happy to change this.

Reply:
Update: I changed this to use the --check flag in both instances, --write should be used by developers if they want to run a script to fix their code automatically.